source/Client/BugTrap.h

```diff
@@ -1,4 +1,4 @@
-/*
+﻿/*
```

Comment:
The encoding of the file has changed.
Most probably, this file was encoded in UTF-8 without BOM and Visual Studio automatically has added BOM to it.

Reply:
revert done.

Reply:
This change is caused by a extension.